frontend/src/stores/userStore.ts

```diff
@@ -12,27 +12,37 @@ export const useUserStore = defineStore('users', {
 		errors: [],
 	}),
 
-  	// getters == computed values
-  	getters: {
-  	  getAllUsers: (state) => state.allUsers,
-  	  getMe: (state) => state.me
-  	},
-  	// actions == methods
-  	actions: {
-    	async login(loginForm: LoginForm) {
-    	  try{
-    	    await postRequest("login", loginForm);
-    	    await this.refreshMe();
-    	    await router.push("/")
-    	  }
-    	  catch (e) {
-    	    alert('Invalid user/password combination')
-    	  }
-	  },
+	// getters == computed values
+	getters: {
+		getAllUsers: (state) => state.allUsers,
+		getMe: (state) => state.me,
+	},
+	// actions == methods
+	actions: {
+		async login(loginForm: LoginForm) {
+			try {
+				await postRequest('login', loginForm);
+				await this.refreshData();
+				await router.push('/');
+			} catch (e) {
+				alert('Invalid user/password combination');
+			}
+		},
+
+		// async getUser(id: number){
+		// 	try{
+		// 		await getRequest('/users/${id}');
+		// 		return User[];
+		// 	}
+		// 	catch (e) {
+		// 		alert('Invalid user');
+		// 	}
+		// },
```

Comment:
Should this still be removed?
Also, if this is going to be used, this should return a single user as it's pinging the findOne controller :)